ext/storage_sources/s3_store/s3_log_system.cpp

```diff
@@ -33,55 +62,48 @@ S3LogSystem::Log(Aws::Utils::Logging::LogLevel logLevel, const char *tag, const
     va_end(args);
 }
 
+// Writes the log stream to the output stream.
 void
 S3LogSystem::LogStream(
   Aws::Utils::Logging::LogLevel logLevel, const char *tag, const Aws::OStringStream &messageStream)
 {
     LogAwsMessage(tag, messageStream.rdbuf()->str().c_str());
 }
 
+// Directs the output stream to WiredTiger's log streams.
 void
 S3LogSystem::LogAwsMessage(const char *tag, const std::string &message) const
 {
     _wtApi->err_printf(_wtApi, NULL, "%s : %s", tag, message.c_str());
 }
 
+// Directs the output stream to WiredTiger's log streams matched at WiredTiger's log stream levels.
 void
 S3LogSystem::LogVerboseMessage(int32_t verbosityLevel, const std::string &message) const
 {
     if (verbosityLevel <= _wtVerbosityLevel) {
-        /* Use err_printf for error and warning messages and use msg_printf for notice, info and
-         * debug messages. */
-        if (verbosityLevel < -1)
+        // Use err_printf for error and warning messages and use msg_printf for notice, info and
+        // debug messages.
+        if (verbosityLevel < WT_VERBOSE_NOTICE)
             _wtApi->err_printf(_wtApi, NULL, "%s", message.c_str());
         else
             _wtApi->msg_printf(_wtApi, NULL, "%s", message.c_str());
     }
 }
 
-void
```

Comment:
I am wondering if we are moving these functions to header file, should we move `S3LogSystem::LogStream` as well. I forgot, what was the logic behind moving these to the header?

Reply:
They were just one line implementations so we moved them to the header (C++ convention?). We have left the functions that are part of the interface and overridden all in the .cpp file so it is easier to follow.